Code-Hammers/code-hammers
# [CHE 41] Create Registration Endpoints

### Description

This ticket is part of [CHE-23](https://codehammers.atlassian.net/browse/CHE-18?atlOrigin=eyJpIjoiMmE3NGU2MWNlYmQ5NDc1NDgxZTE3NDRmMmQ5NmE4OTEiLCJwIjoiaiJ9) Create User Registration story and modifies the existing registration route to work with the graduateInvitation data.

### Jira Task

[CHE-41](https://codehammers.atlassian.net/browse/CHE-41?atlOrigin=eyJpIjoiZmQ3YTI4NzgzMjAxNDlhMmIwZDIwNjkyNTVjOTQyNjciLCJwIjoiaiJ9)

### Testing Instructions

1. Run `dev-ts` script to spin up a local server for Postman testing.
2. Send a variety of requests to ensure that only a properly structured request sent with a valid token results in a new 
    user.

### Checklist


### Additional Notes, Images, etc.

N/A


[CHE-23]: https://codehammers.atlassian.net/browse/CHE-23?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CHE-41]: https://codehammers.atlassian.net/browse/CHE-41?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
